libswc/wscons/atKeynames.h

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
Is this file available from some other package? If possible, I'd prefer if we could just include a header from a dependency.

Reply:
I don't think it's exposed by anything, it's just copied straight from xf86-input-keyboard.

A quick grep of my public X11 headers returns nothing.

Reply:
Okay, that's fine.